webroot/js/participantSummary.js

```diff
@@ -123,7 +148,21 @@ $(document).ready(() => {
 
         url = '/api/partner_config/get';
         doApiCallWithCallback('GET', url, (r) => { loadOptoutWebhooksCallback(r, site.name) }, (err) => { participantSummaryErrorHandler(err, '#webhooksErrorOutput') });
-    
+
+        url = `/api/sharing/keysets/related?site_id=${site.id}&client_types=${site.clientTypes}`;
+        doApiCallWithCallback('GET', url, (r) => { loadRelatedKeysetsCallback(r, site.id, site.clientTypes) }, (err) => { participantSummaryErrorHandler(err, '#relatedKeysetsErrorOutput') });
         $('.section').show();
     });
+
+    $('#doRotateKeysets').on('click', () => {
+        var keysets = $('#relatedKeysetsStandardOutput').text();
+        console.log(keysets);
+        const ja = JSON.parse(keysets);
+        ja.forEach((keyset) => {
+            var url = `/api/key/rotate_keyset_key?min_age_seconds=3600&keyset_id=${keyset.keyset_id}&force=true`;
+//            if ($('#force').is(':checked')) url = url + '&force=true';
```

Comment:
I think the rotation only works when we set `force` to true. Does anyone have any extra insight to this?